source/settings/settings-reference.txt

```diff
@@ -0,0 +1,103 @@
+.. _compass-interface-settings:
+.. _compass-settings-reference:
+
+--------------------------
+Compass Interface Settings
+--------------------------
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The following page provides an overview on options that you can set on the 
+|compass| interface. For more detailed guidance on how to set a specific option, 
+see the task page for the specific option.
+
+Settings
+--------
+
+You can configure the following settings on the |compass| interface:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 30 20 50
+
+   * - Setting 
+     - Category
+     - Definition
+
+   * - Set Read-Only Mode
+     - General
+     - Remove all write and delete capabilities on |compass-short|. 
+
+   * - Enable MongoDB Shell
+     - General
+     - Enable or disable the embedded MongoDB shell on |compass-short|. 
+     
+       If you select :guilabel:`Set Read-Only Mode`, |compass-short| 
+       automatically unchecks the :guilabel:`Enable MongoDB Shell` setting.
```

Comment:
[suggestion] Perhaps, "...Compass automatically disables the MongoDB Shell."

Reply:
Thanks for catching this! Had this copied over from a different page, which required the word "uncheck" for continuity.